### PR TITLE
Fix-Configuring Account Locking due to Failed Login Attempts issues -[5:11:0]

### PR DESCRIPTION
**Pull request by asekawa**
Added extra step for Google Account configuration for the email sending [Less secure app access]

https://is.docs.wso2.com/en/latest/learn/configuring-account-locking-due-to-failed-login-attempts/

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? yes
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: en/docs/learn/configuring-account-locking-due-to-failed-login-attempts.md [hunk 1]
```diff
@@ -14,16 +14,57 @@ Let's learn how Sam implements this!
...
-	```
+	1.	To request password entry from the users who are registered via Management Console, add the following configurations. 
+
+
+	 	```
+	  	[event.default_listener.identity_mgt]
+	  	priority= "50"
+	  	enable = false
+	  	event.default_listener.governance_identity_mgt]
+	    priority= "95"
```
Comment (nilminiwso2) [inline en/docs/learn/configuring-account-locking-due-to-failed-login-attempts.md:27]: This looks misaligned. Does it render properly?

Comment (asekawa) [reply en/docs/learn/configuring-account-locking-due-to-failed-login-attempts.md:27]: Screenshot 2022-05-22 at 21 48 29

**Renders properly and properly aligned in the text-editor as well**

File: en/docs/learn/configuring-account-locking-due-to-failed-login-attempts.md [hunk 2]
```diff
@@ -14,16 +14,57 @@ Let's learn how Sam implements this!
...
+
+		!!! warning "If you are using a Google email account"
+
+			Google has restricted third-party applications and less secure applications from sending emails by default. As WSO2 Identity Server acts as a third-party application when sending emails for password entry, follow the steps below to enable your Google email account to provide access to third-party applications.
+
+			1.	Access [https://myaccount.google.com/security](https://myaccount.google.com/security).
+
+			2.	Under **Signing in to Google** section, turn off the **2-step Verification** option.
+
+			3.  	Enable **Less secure app access** in Google Account security section.
```
Comment (nilminiwso2) [inline en/docs/learn/configuring-account-locking-due-to-failed-login-attempts.md:67]:
```suggestion
			3.  	Enable **Less secure app access** in the Google Account security section.
```

File: en/docs/learn/configuring-account-locking-due-to-failed-login-attempts.md [hunk 3]
```diff
@@ -14,16 +14,57 @@ Let's learn how Sam implements this!
...
 
-	```
-	[event.default_listener.identity_mgt]
-	priority= "50"
-	enable = false
-	[event.default_listener.governance_identity_mgt]
-	priority= "95"
-	enable = true
-	```
+	1.	To request password entry from the users who are registered via Management Console, add the following configurations.
```
Comment (nilminiwso2) [inline en/docs/learn/configuring-account-locking-due-to-failed-login-attempts.md:19]:
```suggestion
	1.	To request password entry from the users who are registered via Management Console, add the following configurations:
```

File: en/docs/learn/configuring-account-locking-due-to-failed-login-attempts.md [hunk 4]
```diff
@@ -14,16 +14,57 @@ Let's learn how Sam implements this!
...
+	 	```toml
+	  	[event.default_listener.identity_mgt]
+	  	priority= "50"
+	  	enable = false
+	  	[event.default_listener.governance_identity_mgt]
+	    priority= "95"
+	   	enable = true
+		```
+
+    2.  To configure the email server to send emails requesting password entry, add the following configurations.
```
Comment (nilminiwso2) [inline en/docs/learn/configuring-account-locking-due-to-failed-login-attempts.md:31]:
```suggestion
    2.  To configure the email server to send emails requesting password entry, add the following configurations:
```

**Review by nilminiwso2 — APPROVED** [review]

**Pull request merged**